pages/linux/sensible-editor.md

```diff
@@ -0,0 +1,19 @@
+# sensible-editor
+
+> Open the default editor.
+
+- Open a file in the default editor:
+
+`sensible-editor %{{file}}`
```

Comment:
Is the percent sign `%` here a typo?

Reply:
no, my misunderstanding of the convention, changed now, thanks